README.md

```diff
@@ -18,6 +19,103 @@ Specifically, if it enables services to scale _during_ deployment.
 > [!TIP]
 > A dashboard combining CloudFormation events, ASG activity and ASG metrics is available here - https://metrics.gutools.co.uk/d/cdvsv1d6vhp1cb/testing-asg-rolling-update.
 
+## Summary
+### Does `AutoScalingRollingUpdate` enable scaling events during deployment?
+No.
+
+When `AutoScalingRollingUpdate` is used, the CloudFormation update will remain in the `UPDATE_IN_PROGRESS` state 
+until the correct number of `SUCCESS` signals have been received.
+
+If more `SUCCESS` signals are received (due to a scale-out mid-deployment) the CloudFormation update will happily continue.
+
+If fewer `SUCCESS` signals are received (due to a scale-in mid-deployment) the CloudFormation update _could_ fail.
+There is no guarantee which instance will be terminated, with the the docs stating:
+
+> Amazon EC2 Auto Scaling features such as instance maintenance policies, termination policies, 
+> and scale-in protection are not available for use with CloudFormation rolling updates. 
+> Plan your rolling updates accordingly.
+> 
+> – https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-updatepolicy.html#cfn-attributes-updatepolicy-rollingupdate
```

Comment:
It's unclear if these restrictions apply to anything that is changing the ASG, or just the CloudFormation update. If termination policies still apply to scale-in, then we could implement a [custom termination policy](https://docs.aws.amazon.com/autoscaling/ec2/userguide/lambda-custom-termination-policy.html), as we'd need to choose an instance such that it doesn't negatively impact the healthy host count of the load balancer, which none of the [pre-defined ones](https://docs.aws.amazon.com/autoscaling/ec2/userguide/ec2-auto-scaling-termination-policies.html#predefined-termination-policies) do.